apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -3,20 +3,20 @@
 
 $card-min-width: 270px;
 $card-image-height: 180px;
-$card-gap: f.$spacing-e;
-$card-container-max-width: calc(($card-min-width * 3) + ($card-gap * 2));
+$card-gap: var(--dt-spacing-e);
+$card-container-max-width: calc(($card-min-width * 3) + 48px); // 48px is card-gap * 2
```

Comment:
Doesnt calc work for `$card-gap`?

Reply:
I think we'd need to convert the variable to be a CSS variable (but we should do this imo and phase out the SCSS variables where possible)

Reply:
@fernandofranca when I changed the value of `$card-gap` to use a css variable, the calculation of `card-container-max-width` didn't work anymore 👀 i assume it's related to the unit conversions but I'm not 100% sure. maybe @ashleynolan would know?

i will also address scss variables cleanup in my follow up PR if you don't mind.

Reply:
Let me know if you have issues with this in the followup – can help you make the change 😄